static/js/client.js

```diff
@@ -50,6 +50,7 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
     , delta = Nightscout.plugins('delta')
     , direction = Nightscout.plugins('direction')
     , errorcodes = Nightscout.plugins('errorcodes')
+    , translate = Nightscout.language.translate
```

Comment:
related to previous comment, here we could do `translate = Nightscout.language(serverSettings.defaults.language).translate` instead of using `serverSettings` in the bundle